### PR TITLE
benchmark: improve wording

### PR DESCRIPTION
**Pull request by cofyc**
### What is changed, added or deleted? 

This PR improves wording in `TiDB in Kubernetes Sysbench 性能测试.md`.

This PR is **still in progress** and more commit will be pushed to improve the language of this document.

### Which version does your change affect? 

`dev`, `v3.0`

### CODE REVIEW

File: dev/benchmark/sysbench-in-k8s.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ set global tidb_disable_txn_auto_retry=0;
 | TiDB     | c2-standard-16 | 3     |
 | Sysbench | c2-standard-30 | 1     |
 
-在多可用区相对单可用区性能对比中，因为测试时 (2019.08) GCP 的 Region 均没有 3 个可用区可同时提供 c2 机器，我们选择了如下机器型号进行测试：
+分别在多可用区和单可用区中对 TiDB 进行性能测试，并将结果相对比。测试时 (2019.08)，一个 GCP 区域 (Region) 下，不存在三个能同时提供 c2 机器的可用区，我们选择了如下机器型号进行测试：
```
Comment (TomShawn) [inline dev/benchmark/sysbench-in-k8s.md]:
```suggestion
分别在多可用区和单可用区中对 TiDB 进行性能测试，并将结果相对比。在测试时 (2019.08)，一个 GCP 区域 (Region) 下不存在三个能同时提供 c2 机器的可用区，所以我们选择了如下机器型号进行测试：
```

File: v3.0/benchmark/sysbench-in-k8s.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ set global tidb_disable_txn_auto_retry=0;
 | TiDB     | c2-standard-16 | 3     |
 | Sysbench | c2-standard-30 | 1     |
 
-在多可用区相对单可用区性能对比中，因为测试时 (2019.08) GCP 的 Region 均没有 3 个可用区可同时提供 c2 机器，我们选择了如下机器型号进行测试：
+分别在多可用区和单可用区中对 TiDB 进行性能测试，并将结果相对比。测试时 (2019.08)，一个 GCP 区域 (Region) 下，不存在三个能同时提供 c2 机器的可用区，我们选择了如下机器型号进行测试：
```
Comment (TomShawn) [inline v3.0/benchmark/sysbench-in-k8s.md]:
```suggestion
分别在多可用区和单可用区中对 TiDB 进行性能测试，并将结果相对比。在测试时 (2019.08)，一个 GCP 区域 (Region) 下不存在三个能同时提供 c2 机器的可用区，所以我们选择了如下机器型号进行测试：
```

**Comment by cofyc** [comment]
@TomShawn can this be merged now?

**Comment by TomShawn** [comment]
@yikeke PTAL, thanks!

File: dev/benchmark/sysbench-in-k8s.md [hunk 3]
```diff
@@ -112,11 +112,11 @@ GKE 网络模式使用具备更好扩展性以及功能强大的 [VPC-Native](ht
 
 #### CPU
 
-在单可用集群测试中，我们为 TiDB/TiKV 选择 c2-standard-16 机型测试。在单可用与多可用集群的对比测试中，因为 GCP 没有区域可以同时在 3 个可用区里申请 c2-standard-16 机型。我们选择了 n1-standard-16 机型测试。
+在单可用集群测试中，我们为 TiDB/TiKV 选择 c2-standard-16 机型测试。在单可用与多可用集群的对比测试中，因为GCP 区域 (Region) 下不存在三个能同时申请 c2-standard-16 机型的可用区，所以我们选择了 n1-standard-16 机型测试。
```
Comment (yikeke) [inline dev/benchmark/sysbench-in-k8s.md]:
```suggestion
在单可用集群测试中，我们为 TiDB/TiKV 选择 c2-standard-16 机型测试。在单可用与多可用集群的对比测试中，因为 GCP 区域 (Region) 下不存在三个能同时申请 c2-standard-16 机型的可用区，所以我们选择了 n1-standard-16 机型测试。
```

File: v3.0/benchmark/sysbench-in-k8s.md [hunk 4]
```diff
@@ -112,11 +112,11 @@ GKE 网络模式使用具备更好扩展性以及功能强大的 [VPC-Native](ht
 
 #### CPU
 
-在单可用集群测试中，我们为 TiDB/TiKV 选择 c2-standard-16 机型测试。在单可用与多可用集群的对比测试中，因为 GCP 没有区域可以同时在 3 个可用区里申请 c2-standard-16 机型。我们选择了 n1-standard-16 机型测试。
+在单可用集群测试中，我们为 TiDB/TiKV 选择 c2-standard-16 机型测试。在单可用与多可用集群的对比测试中，因为GCP 区域 (Region) 下不存在三个能同时申请 c2-standard-16 机型的可用区，所以我们选择了 n1-standard-16 机型测试。
```
Comment (yikeke) [inline v3.0/benchmark/sysbench-in-k8s.md]:
```suggestion
在单可用集群测试中，我们为 TiDB/TiKV 选择 c2-standard-16 机型测试。在单可用与多可用集群的对比测试中，因为 GCP 区域 (Region) 下不存在三个能同时申请 c2-standard-16 机型的可用区，所以我们选择了 n1-standard-16 机型测试。
```

**Comment by TomShawn** [comment]
> Rest LGTM. After you resolve the two comments, feel free to merge this PR.

Thanks!

**Pull request merged**